py_ecc/bls_api/typing.py

```diff
@@ -0,0 +1,20 @@
+# This module will not be included in the PR.
+# These types should be replaced with those in eth-typing
+
+from typing import (
+    NewType,
+    Tuple,
+)
+
+from py_ecc.fields import (
+    optimized_bls12_381_FQ as FQ,
+    optimized_bls12_381_FQ2 as FQ2,
+)
+
+
+# TODO: FQ and FQ2 here are invalid types, they need to be fixed in the future
```

Comment:
Due to the recent refactor of field elements in py_ecc, the type check is not available for the time being.
@Bhargavasomu Is there a workaround before the refactor is complete?

Reply:
@ChihChengLiang the type checking has been removed for the rest of the modules for now. But the `type checking` can be enabled only for this particular `bls_api` through `mypy`. As far as the `type hints` are concerned, most of them should be covered by using `py_ecc/typing.py`. But I have removed `type checking` temporarily till the refactor is complete.